config/initializers/elasticsearch.rb

```diff
@@ -1,6 +1,13 @@
+Rails.logger.info("Initializing Elasticsearch client")
+
+# FIXME: switch to gem?
+# https://github.com/omc/bonsai-elasticsearch-rails/blob/master/lib/bonsai/elasticsearch/rails/railtie.rb
```

Comment:
I didn't use the client because I figured here we can somehow edit so that we don't need to be running ES during unit tests...